ci/infra/testrunner/tests/conftest.py

```diff
@@ -81,3 +81,60 @@ def kubectl(conf, target):
 def platform(conf, target):
     platform = platforms.get_platform(conf, target)
     return platform
+
+
+@pytest.fixture
+def cluster_ready(kubectl):
+    def func():
+        wait(kubectl.run_kubectl, 'wait --for=condition=Ready node --all --timeout=1m', wait_delay=60, wait_timeout=300, wait_backoff=30, wait_retries=5)
+        wait(kubectl.run_kubectl, 'wait --timeout=1m --for=condition=Ready pods --all --namespace=kube-system --field-selector=status.phase!=Succeeded', wait_delay=60, wait_timeout=300, wait_backoff=30, wait_retries=5, wait_allow=(RuntimeError))
```

Comment:
Is the wait_delay required for the second wait? I find it a bit strange but probably there is a good reason.

Reply:
I'll try it without it